frontend/tests/e2e/navbar-mobile.test.ts

```diff
@@ -0,0 +1,67 @@
+import { expect, test } from '@nuxt/test-utils/playwright'
+import { devices } from '@playwright/test'
+
+/**
+ * Regression test to verify that the navbar remains visible in the viewport
+ * when the page is scrolled to the bottom on mobile devices.
+ *
+ * This test:
```

Comment:
This list is unnecessary, also the description above is not required, because we have test name